pkg/trace/api/api.go

```diff
@@ -329,19 +308,27 @@ func (r *HTTPReceiver) tagStats(req *http.Request) *info.TagStats {
 	})
 }
 
-func (r *HTTPReceiver) decodeTraces(v Version, req *http.Request) (pb.Traces, error) {
-	if v == v01 {
+func (*HTTPReceiver) decodeTraces(v Version, req *http.Request) (pb.Traces, error) {
```

Comment:
Suggestion: we may want to monitor/track which version of the Agent endpoints are being used.
It'd be helpful for troubleshoot but also global monitoring of users transitioning to newest versions.

Maybe add a tag to the existing `datadog.trace_agent.receiver.*` metrics?

Reply:
We can, I will do this immediately in a follow-up PR because I feel it's a bit unrelated and I don't want to derail this. I hope that's ok with you. I'm going to start now and base on this one.

Reply:
Update: https://github.com/DataDog/datadog-agent/pull/6122